src/main/scala/org/deepdive/ddlog/DeepDiveLogCompiler.scala

```diff
@@ -584,6 +609,7 @@ class QueryCompiler(cq : ConjunctiveQuery, hackFrom: List[String] = Nil, hackWhe
         "input_query" -> QuotedString("\n"+inputQueries.mkString(" UNION ALL ")+"\n"),
         "function" -> QuotedString(func),
         "weight" -> QuotedString(weight),
+        "non_category_weight_cols" -> nonCategoryWeightCols.toList.sorted,
```

Comment:
Nice.  In the future, we should drop the `weight` string, get rid of crappy parsing w/ regex from DD, and just produce a completely structured intermediate representation.